packages/drivers/odsp-driver/src/odspFluidFileLink.ts

```diff
@@ -61,11 +61,12 @@ export function encodeOdspFluidDataStoreLocator(locator: OdspFluidDataStoreLocat
 function decodeOdspFluidDataStoreLocator(
 	encodedLocatorValue: string,
 	siteOriginUrl: string,
+	requireFluidSignature: boolean = true,
```

Comment:
I'm not super familiar with these APIs, so someone else would be better suited to review the changes from a functional perspective. But if we're adding new parameters, let's be sure to add `@param` comments for them where appropriate and document what they mean and what their defaults are.

Reply:
Good idea, I'll add it to the exported function